roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -93,3 +93,17 @@
     name: "{{ ( (ansible_distribution_major_version | int) < 8) | ternary('libselinux-python','python3-libselinux') }}"
     state: present
   become: true
+
+- name: Stop service firewalld, if started
+  service:
```

Comment:
you can write this as 1 task no ? for sure with `systemd:` you can

Reply:
Are you saying like
```
- name: Disable service firewalld
  systemd:
      name: firewalld
      state: stopped
      enabled: no
```

Reply:
Thanks for your advice, let me try and test it.

Reply:
confirmed that works, nice advice :)

Reply:
you are welcome